simpledotorg/simple-server
# Get quarterly cohort numbers from v2 schema when enabled

**Story card:** [ch5871](https://app.shortcut.com/simpledotorg/story/5871/grab-cohort-data-from-reporting-pipeline)

## Because

We have the cohort numbers in the reporting pipeline but need to grab them and display them for reports

## This addresses

showing reports from the pipeline for cohort

## Test instructions

check out the branch, view cohort reports, cycle between v1 and v2 via query string to compare
